pkg/gardenlet/bootstrap/util/util_test.go

```diff
@@ -64,13 +66,13 @@ var _ = Describe("Util", func() {
 				Organization: []string{organization},
 				CommonName:   "test-cn",
 			}
-			digest, err := DigestedName(signer.Public(), subject, []certificatesv1.KeyUsage{certificatesv1.UsageDigitalSignature})
+			digest, err := csr.DigestedName(signer.Public(), subject, []certificatesv1.KeyUsage{certificatesv1.UsageDigitalSignature}, bootstrap.SeedCSRPrefix)
```

Comment:
We usually avoid usages of constants from the production code in the unit tests:
```suggestion
			digest, err := csr.DigestedName(signer.Public(), subject, []certificatesv1.KeyUsage{certificatesv1.UsageDigitalSignature}, "seed-csr-")
```

or even we can use any prefix

```suggestion
			digest, err := csr.DigestedName(signer.Public(), subject, []certificatesv1.KeyUsage{certificatesv1.UsageDigitalSignature}, "foo-csr-")
```